code/tick/feed.q

```diff
@@ -84,4 +84,4 @@ init:{
 h:.servers.gethandlebytype[`tickerplant;`any]
 
 init 0
-.z.ts:feed
+.timer.repeat[.proc.cp[];0Wp;0D00:00:00.200;(`feed;`);"Publish Feed"];
```

Comment:
this will add the repeating job in mode 2 by default, think that's the right mode to use here?  (I'm not really sure, genuinely asking)